cluster/gce/gci/configure-helper.sh

```diff
@@ -1410,6 +1410,10 @@ ExecStart=${kubelet_bin} \$KUBELET_OPTS
 WantedBy=multi-user.target
 EOF
 
+  if [[ ${ENABLE_CREDENTIAL_SIDECAR:-false} == "true" ]]; then
```

Comment:
We should consider splitting enabling from configuring. This is ok for alpha but later it will make sense to be able to configure without enabling.

Reply:
Gotcha - would an additional `CONFIGURE_CREDENTIAL_SIDECAR` flag make sense in that case?

Reply:
Yes, not critical. Also if you go this route you would want to control it of the same kube-up CLI parameter.
export CONFIGURE_CREDENTIAL_SIDECAR="${KUBE_ENABLE_CREDENTIAL_SIDECAR:-false}"